cls/SourceControl/Git/Utils.cls

```diff
@@ -262,6 +262,7 @@ ClassMethod UserAction(InternalName As %String, MenuName As %String, ByRef Targe
         set Action = 7
         quit $$$OK
     } elseif (menuItemName = "Commit") {
+        do ..CheckUserIdentity()
```

Comment:
We can do better than outputting warnings - if user identity is not configured, the commit *will* fail.

See https://docs.intersystems.com/irislatest/csp/documatic/%25CSP.Documatic.cls?LIBRARY=%25SYS&CLASSNAME=%25Studio.Extension.Base#UserAction - we should display an alert dialog rather than a prompt if the identity isn't configured. We should do the same for the Sync menu item as it will also subsequently fail.